.github/workflows/deploy-windows.yml

```diff
@@ -0,0 +1,110 @@
+name: deploy-windows
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
for prereleases I use `p{date}` as tag.

Reply:
The binaries will be added after the prerelease is created, so you can set the `p{date}' yourself. If we want to create a completely automatic prerelease, we need to add a `Prerelease.md` file containing all the changes.